modules/infra/control/port.c

```diff
@@ -95,8 +95,9 @@ int port_configure(struct iface_info_port *p, uint16_t n_txq_min) {
 	if ((ret = rte_eth_dev_info_get(p->port_id, &info)) < 0)
 		return errno_log(-ret, "rte_eth_dev_info_get");
 
-	if (strcmp(info.driver_name, "net_tap") == 0) {
-		p->n_txq = RTE_MAX(p->n_txq, p->n_rxq);
```

Comment:
This patch has nothing to do with the series about larger packets.

In any case, I am not following.
Why remove this part above?

With the previous code, if a user requests more rxq, this would result in unneeded txq, but I don't see a problem with this.

Reply:
I'll drop this patch for now.